docs/src/templates/includes/components/core/menu/status/je-menu-status-delivery-open.hbs

```diff
@@ -1,5 +1,5 @@
-<p class="c-badge c-badge--info c-badge--rounded l-centered">
+<span class="c-badge c-badge--info c-badge--rounded c-badge--alignCenter">
```

Comment:
What's the reason to change these to `span`'s?

Not sure if there are any accessibility implications for just using spans to wrap text (so no semantic tag wrapped around it)…?

Reply:
It is wrapped in a `p` tag in the template which pulls this one in, unfortunately it's hard to tell by way the partials are separated out.

![screen shot 2018-09-14 at 09 14 15](https://user-images.githubusercontent.com/89059/45539105-ac001f00-b800-11e8-99bd-d8df9d65dede.png)

Reply:
https://github.com/justeat/global-component-library/pull/58/files#diff-43ffbb6c9652e23638fa60577b748b76R44